runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/actions/zip.ts

```diff
@@ -1,11 +1,15 @@
 import englishZipTranslations from '../../../en/process/actions/zip';
 
-const zipTranslations: typeof englishZipTranslations = {
-    'Process.Details.Modeler.Actions.Zip.UnzipFile.Label': 'Rozpakuj archiwum',
-    'Process.Details.Modeler.Actions.Zip.Unzip.Path': 'Ścieżka',
-    'Process.Details.Modeler.Actions.Zip.Unzip.Name': 'Nazwa',
+const zipTranslations: typeof englishZipTranslations  = {
+    'Process.Details.Modeler.Actions.Zip.Unzip.Label': 'Rozpakuj archiwum',
+    'Process.Details.Modeler.Actions.Zip.Path': 'Ścieżka',
+    'Process.Details.Modeler.Actions.Zip.Name': 'Nazwa',
     'Process.Details.Modeler.Actions.Zip.Unzip.Path.Info': 'Ścieżka do folderu zawierającego plik zip, np. "C:\\Users\\User\\Documents".',
-    'Process.Details.Modeler.Actions.Zip.Unzip.Name.Info': 'Nazwa pliku (bez rozszerzenia), np. "zippedArchive".'
+    'Process.Details.Modeler.Actions.Zip.Unzip.Name.Info': 'Nazwa pliku (bez rozszerzenia), np. "zippedArchive".',
+    'Process.Details.Modeler.Actions.Zip.Zip.Label': 'Stwórz archiwum',
+    'Process.Details.Modeler.Actions.Zip.Zip.Path.Info': 'Ścieżka do folderu, który ma zostać zarchiwizowany, np. "C:\\Users\\User\\Documents\\folderDoArchiwizacji',
```

Comment:
and here should be: 
'Ścieżka do folderu lub znajdującego się w nim pliku, który ma zostać zarchiwizowany, np. "C:\\Users\\User\\Documents\\folderDoArchiwizacji',